jetty-core/jetty-server/src/main/config/etc/jetty-dos.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="insertHandler">
+    <Arg>
+      <New id="DosHandler" class="org.eclipse.jetty.server.handler.DoSHandler">
+        <Arg name="getId">
+          <Get>
+            <Name><Property name="jetty.dos.id.type" default="ID_FROM_REMOTE_ADDRESS"/></Name>
+            <Class><Property name="jetty.dos.id.class" default="org.eclipse.jetty.server.handler.DoSHandler"/></Class>
```

Comment:
I'd swap these 2 lines, so it looks more Java like as in `DoSHandler.ID_FROM_REMOTE_ADDRESS`.

Reply:
Unfortunately that order does not validate again the DTD